symforce/codegen/codegen_util.py

```diff
@@ -342,20 +342,19 @@ def get_formatted_list(
             formatted_symbols = [sm.Symbol(key)]
             flattened_value = [value]
         elif issubclass(arg_cls, geo.Matrix):
-            if isinstance(config, codegen_config.PythonConfig):
-                # TODO(nathan): Not sure this works for 2D matrices
+            if config.matrix_is_1d:
```

Comment:
I preserved the behavior, but don't fully understand it. Would like to discuss

Reply:
Let's move this discussion into the IMU preintegration review (#160) where it is in play

Reply:
resolved in the other PRe